### ISSUE TITLE
Project Idea

### ISSUE THREAD

**Issue by jnlon** [emilyhorsman/haskell-resources#6]
We should write a clone of macalytics in Haskell and call it "hackalytics" 😄

**Comment by emilyhorsman** [comment]
It is written in Haskell!

**Comment by jnlon** [comment]
Aww, I assumed nodejs + elm

Might be fun to rewrite it though (and fix that horrible UI too!)

**Comment by emilyhorsman** [comment]
Yeah I will say I am more interested in rewriting the front-end than the back
